tests/fixtures/analog/uplinkCommands.js

```diff
@@ -0,0 +1,102 @@
+export const uplinkSimpleCommands = {
```

Comment:
there is no difference between the commands in the structures and checks, so it looks redundant to use "simple" or "complex" 

Also "uplinkCommands.js" with `export const uplink...` - too much, maybe just `export default []` ?

Reply:
I added complex commands, now there is a difference in the structures